src/BatchFormat/source.extension.vsixmanifest

```diff
@@ -15,19 +15,25 @@ Function: Remove, Sort, Using, Format Document, and more combinations.</Descript
     <InstalledByMsi>false</InstalledByMsi>
     <SupportedProducts>
       <VisualStudio Version="10.0">
-        <Edition>Ultimate</Edition>
         <Edition>Premium</Edition>
         <Edition>Pro</Edition>
+        <Edition>Ultimate</Edition>
       </VisualStudio>
       <VisualStudio Version="11.0">
-        <Edition>Ultimate</Edition>
         <Edition>Premium</Edition>
         <Edition>Pro</Edition>
+        <Edition>Ultimate</Edition>
       </VisualStudio>
       <VisualStudio Version="12.0">
-        <Edition>Ultimate</Edition>
+        <Edition>Community</Edition>
         <Edition>Premium</Edition>
         <Edition>Pro</Edition>
+        <Edition>Ultimate</Edition>
+      </VisualStudio>
+      <VisualStudio Version="14.0">
+        <Edition>Community</Edition>
+        <Edition>Enterprise</Edition>
+        <Edition>Pro</Edition>
       </VisualStudio>
```

Comment:
@yongfa365, can you please test with this change: https://github.com/am11/BatchFormat/commit/05cc1f7b9a8d2326aaa2a9a009fcbfa0ee7d7864

I would be better if we figure out a better way that doesn't result in bundling the SDK dll within the package.

In cmd with `git` in `PATH`:

```cmd
git clone https://github.com/am11/BatchFormat
cd BatchFormat
%ProgramFiles(x86)%\MSBuild\12.0\Bin\MSBuild .\src\BatchFormat.sln
:: change 12.0 with the version of VS you want to test with
```

Please let me know if there is still an error.